CHANGELOG.md

```diff
@@ -14,7 +14,7 @@
 
 * Fix: Avoid open-ended dependencies for sinatra-contrib and rack-protection ([#1949])
 
-* Fix: Helpful message when `Sinatra::Runner` times out ([#1975])
+* Fix: Helpful message when `Sinatra::Runner` times out ([#1960])
```

Comment:
This is not correct

Reply:
This did happen in #1975 at https://github.com/sinatra/sinatra/pull/1975/files#diff-27e0daa68dc114ddbf9c51bb075e3ca12a107537259d742b5309e6d9e821e86f

Reply:
Ah my mistake. 1960 looked related to `Sinatra`, so I thought there we actually want that link.

Reply:
Fixed.